src/Sarif/Schemata/sarif-schema.json

```diff
@@ -717,13 +717,13 @@
           "type": "integer"
         },
 
-        "ruleConfiguration": {
-          "description": "An array of rule configuration objects that describe runtime analysis behavior.",
+        "outputConfigurationOverrides": {
+          "description": "An array of notification configuration override objects that describe runtime reporting behavior.",
```

Comment:
>notification [](start = 38, length = 12)

"notification configuration" => "output configuration" (assuming we keep that name).

Reply:
Hah, you used a good word in the description! What do you think of `reportingDescriptor`, `reportingConfiguration`, and `reportingConfigurationOverride`?

---
In reply to: [256191271](https://github.com/Microsoft/sarif-sdk/pull/1280#discussion_r256191271) [](ancestors = 256191271)

Reply:
I really like those names. Just letting you know it's not just a quibble. I really didn't like "outputXxx", and I jumped when I saw the word "reporting".

---
In reply to: [256191513](https://github.com/Microsoft/sarif-sdk/pull/1280#discussion_r256191513) [](ancestors = 256191513,256191271)

Reply:
great, i like it too. reportingDescriptor it is. will push an update.

---
In reply to: [256191779](https://github.com/Microsoft/sarif-sdk/pull/1280#discussion_r256191779) [](ancestors = 256191779,256191513,256191271)